gsa/src/web/components/form/multiselect.js

```diff
@@ -176,6 +176,7 @@ class MultiSelect extends React.Component {
       items,
       menuPosition = 'adjust',
       width = DEFAULT_WIDTH,
+      grow = '0',
```

Comment:
Not sure if grow should get a default value? @swaterkamp what do you think?

Reply:
the default value of 0 doesn't do anything as far as i can tell, so if we want the old appearance, setting it to 0 will give us that. I think only positive integers make a difference.

https://stackoverflow.com/questions/28473807/how-does-flex-grow0-get-interpreted

Reply:
We could set a default of 0 in the SelectContainer, where it actually will be applied in the end. I find NOT growing to be the right option as default and just explicitly setting it to 1 if we REALLY want it.